src/translations/nbTranslations.json

```diff
@@ -1622,7 +1622,8 @@
       "pages_to": "til",
       "place_for_event": "Sted for arrangement",
       "print_issn": "Print ISSN",
-      "protected_type": "Du mangler tilgang. Kontakt institusjonen som har gitt deg graden for å få den publisert. Om den er utenlandsk, be om kuratorstøtte for hjelp til publiseringen.",
+      "protected_degree_type": "Du mangler tilgang. Kontakt institusjonen som har gitt deg graden for å få den publisert. Om den er utenlandsk, be om kuratorstøtte for hjelp til publiseringen.",
```

Comment:
NB! Husk å rename denne i Lokalise før merge, for å ta vare på engelsk og nynorsk oversettelser

Reply:
Er det fila eller publikasjonen som er utenlandsk man skal kontakte kurator om?

Reply:
Institusjonen (hvor man har tatt graden). Feks om jeg som nordmann tar en grad ved Universitetet i Madrid, og ønsker å legge inn det i NVA. Mest akutelt for doktorgrader vil jeg tro.
Denne vises altså som info for å blokkere brukere i å velge `Degree`-typer, da dette bare skal kunne registreres av bestemte kuratorer